.travis.yml

```diff
@@ -1,26 +1,28 @@
 language: node_js
 node_js: '10'
 cache:
+  yarn: true
   directories:
-    - ~/.npm
+    - node_modules
```

Comment:
Caching `node_modules` is not safe. `node_modules` can contain binaries that are downloaded and compiled in postinstall scripts, and these binaries depend on the environment. Caching `~/.cache`/`~/.npm` is the proper way to do it (and if that cache is hydrated, install is nothing but a `cp -r`)

Reply:
I was following https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn 

I don't see anything related to npm or yarn in my `~/.cache` (macOS). Is this different for Travis (probably Linux)?

Feel free to push to this PR with improvements!

Reply:
ping

Reply:
Removed caching for `node_modules`